packages/webapp/src/views/credits-purchase/check-and-submit.html

```diff
@@ -57,7 +57,7 @@
     'value': credits.amounts.a1,
     'valueDataTestId': 'credits-a1-value',
     'valueId': 'credits-a1-value',
-    'href': credits.changeUrl,
+    'href': credits.changeUrl + '?focus=a1-units',
```

Comment:
I have just tested and this works perfect without any js. Please try and let me know

` 'href': credits.changeUrl+'#a1-units',`

Reply:
Yes, Kiran, your suggestion should work. However, there's another issue: using 'href': credits.changeUrl+'#a1-units' will focus on the element with scrolling. If the user needs to confirm which box it is, they will need to scroll up!. I will demonstrate this tomorrow.